test/python/test_op.py

```diff
@@ -452,3 +452,52 @@ def test_multiple_outputs():
 
     assert 1 - r2_score(result0.detach().numpy().flatten(), ref1.flatten()) < 0.01
     assert 1 - r2_score(result1.detach().numpy().flatten(), ref2.flatten()) < 0.01
+
+
+@pytest.mark.parametrize("batch", [16, 128])
+@pytest.mark.parametrize("hidden_dim", [128, 256])
+@pytest.mark.parametrize("exponent", ["tensor", "float"])
+def test_power(batch, hidden_dim, exponent):
```

Comment:
Can youo add a test where the exponent is a `model.parameter` ?

Reply:
Added functionality and additional test for tensor parameters for power op